tests/integration/ha_tests/test_upgrade_from_stable.py

```diff
@@ -76,10 +58,11 @@ async def test_deploy_stable(ops_test: OpsTest) -> None:
     assert len(ops_test.model.applications[DATABASE_APP_NAME].units) == 3
 
 
-@pytest.mark.group(1)
 @pytest.mark.abort_on_fail
 async def test_pre_upgrade_check(ops_test: OpsTest) -> None:
     """Test that the pre-upgrade-check action runs successfully."""
+    # TODO remove once we release to stable
```

Comment:
:+1:  should we have ticket here? Or prepared PR?

We can promote the current 16/edge to 16/beta and use it here.

Reply:
A prepared PR pointing to 16/beta would be fine. So we can test it before we promote to stable.

Reply:
We can promote with the dispatch flows to test that they work once we merge.